ImplementationGuide/markdown/Datenobjekte/Operations.md

```diff
@@ -16,15 +16,15 @@ Generell wird darauf hingewiesen, dass abhängig davon, welcher Client oder Benu
 
 User Story für die folgenden Beispiele: Ein Patient bucht über ein externes Patientenportal einen Termin in der allgemeinmedizinischen Ambulanz eines Krankenhauses. Da der Patient seit Tagen Bauchschmerzen hat, die in den letzten Stunden stärker werden, wählt er die Priorität "Notfall".
 
-1. Abfrage aller Kodierungen der Behandlungsleistungen durch den Termin-Requestor: `GET https://example.org/fhir/CodeSystem?context-type-value=https://gematik.de/fhir/isik/v3/Terminplanung/CodeSystem/ContextType|ResourceUsage$http://hl7.org/fhir/resource-types|HealthcareService`
+1. Abfrage aller Kodierungen der Behandlungsleistungen durch den Termin-Requestor: `GET https://example.org/fhir/CodeSystem?context-type-value=https://gematik.de/fhir/isik/v3/Terminplanung/CodeSystem/ContextType|ResourceUsage$http://hl7.org/fhir/resource-types|HealthcareService` bzw. `GET https://example.org/fhir/CodeSystem?context-type-value=https://gematik.de/fhir/isik/v3/Terminplanung/CodeSystem/ContextType|ResourceUsage$http://hl7.org/fhir/resource-types|Schedule`
 
-Das Termin-Repository MUSS alle CodeSysteme exponieren, welche für die Kodierung eines HealthcareService relevant sind. Die Anfrage ist nicht auf HealthcareService beschränkt, weitere Ressourcen-Kontexte können abgefragt werden. Alle verwendeten CodeSysteme MÜSSEN exponiert werden, insoweit diese sich als CodeSystem-Ressourcen ausdrücken lassen.
+Das Termin-Repository MUSS alle CodeSysteme exponieren, welche für die Kodierung einer verfügbaren Behandlungsleistung relevant sind. Die Anfrage ist nicht auf HealthcareService beschränkt, weitere Ressourcen-Kontexte können hierfür relevant sein. Beispielsweise erfolgt die Kodierung der Behandlungsleistung für einen Behandler im Kontext eines Schedules (vgl. {{pagelink:ImplementationGuide/markdown/Datenobjekte/ISiKKalenderSchedule.md, text:Schedule.serviceType}}). Alle verwendeten CodeSysteme MÜSSEN exponiert werden, insoweit diese sich als CodeSystem-Ressourcen ausdrücken lassen.
```

Comment:
"Behandlungsleistung für einen Behandler" würde ich ersetzen mit "gesundheitlichen Dienstleistungen" (entspricht auch eher dem default VS) ... hier kann ja alles von diagnostischen Leistungen , Sehtest , Yoga etc. dabeisein

Reply:
Beides, einzelne Ärzte können direkt über eine Referenz von Schedule -> Practitioner abgebildet werden für den Fall, dass eine Leistung nur durch eine bestimmte Person angeboten werden kann. In diesem Fall erfolgt die Kodierung der Leistungen als Teil des Schedules. Für alles weitere erfolgt die Kodierung innerhalb eines HealthcareService.